optimum/exporters/openvino/model_configs.py

```diff
@@ -58,6 +58,7 @@
     DummyVisionInputGenerator,
     FalconDummyPastKeyValuesGenerator,
     MistralDummyPastKeyValuesGenerator,
+    GemmaDummyPastKeyValuesGenerator,
```

Comment:
making code style checking happy
```suggestion
    GemmaDummyPastKeyValuesGenerator,
    MistralDummyPastKeyValuesGenerator,
```